### PR TITLE
Child theme

### PR DESCRIPTION
**Pull request by spence-man**
[Staging Site](https://chappress-staging.chapman.edu/)
[Trello Story](https://trello.com/c/ZgM5kFHY/3-as-a-user-i-want-the-chappress-site-to-have-a-customized-version-of-the-default-wordpress-theme)

### CODE REVIEW

File: content/themes/ChapPress/functions.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?php
```
Comment (tatwell) [inline content/themes/ChapPress/functions.php]: What are these binary files above? Do we need all these `.sass-cache` files?

Comment (tatwell) [reply content/themes/ChapPress/functions.php]: Also, no camel case for file/folder names. (Unless this is a WP convention.) Please rename this directory `chappress` or `chap-press`. `chappress` is fine.

File: content/themes/ChapPress/sass/_entry.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.site-content-contain {
```
Comment (tatwell) [inline content/themes/ChapPress/sass/_entry.scss]: Are these binary files used by the webserver? Why in 2 places?

**Review by tatwell — CHANGES_REQUESTED** [review]
Can you rename the directory then issue a new PR and close this one?

**Comment by tatwell** [comment]
Can we use logo and favicon from the wiki site?

https://wimops.chapman.edu/wiki/Main_Page

**Comment by spence-man** [comment]
Yes, added to .gitgnore. Still there but working on a solution.

File: content/themes/ChapPress/functions.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+/* Logo 
+   ######################################################
+   https://codex.wordpress.org/Customizing_the_Login_Form
+*/
+
+function my_login_logo() { ?>
+    <style type="text/css">
+        #login h1 a, .login h1 a {
+            background-image: url(<?php echo get_stylesheet_directory_uri(); ?>/images/site-login-logo.svg);
```
Comment (tatwell) [inline content/themes/ChapPress/functions.php]: Does this need to be changed?

**Pull request closed without merging**